Cargo.toml

```diff
@@ -54,7 +54,7 @@ members = ["rand_core"]
 bincode = "1.0"
 
 [target.'cfg(target_os = "cloudabi")'.dependencies]
-cloudabi = "0.0.3"
+cloudabi = {version="0.0.3", optional=true}
```

Comment:
I think these dependencies are actually required on these platforms?

Reply:
Yes, but so is `winapi` on Windows, and `libc` on Unix, which are also optional.

Reply:
True; then `std` needs to depend on them.

Do you want to put all the platform-specific dependencies in the same part of the file?